pkg/nsx/services/firewall.go

```diff
@@ -599,7 +1177,7 @@ func (service *SecurityPolicyService) CreateOrUpdateSecurityPolicy(obj *v1alpha1
 	} else {
 		err = service.createOrUpdateGroups(groupsClient, nsxGroups)
 		if err != nil {
-			log.Error(err, "failed to create or update groups", "nsxSecurityPolicy.Id", nsxSecurityPolicy.Id)
+			log.Error(err, "failed to create or update group", "nsxSecurityPolicy.Id", nsxSecurityPolicy.Id)
```

Comment:
I think it should be `groups`, not `group`. I think this case is NOT similar to my previous comment https://github.com/vmware-tanzu/nsx-operator/pull/38#discussion_r793307770.

Reply:
Thanks for explain.